src/networking.c

```diff
@@ -4787,6 +4831,11 @@ void clientKillCommand(client *c) {
         if (parseClientFiltersOrReply(c, i, &client_filter) != C_OK) {
             /* Free the intset on error */
             goto client_kill_done;
+        } else {
+            /* No skipme option provided, default behavior with the new syntax is to skip */
+            if (client_filter.skipme == -1) {
+                client_filter.skipme = 1;
+            }
```

Comment:
This is not necessary. It's initialized to `1` 15 lines above this.

Reply:
This is unnecessary after the last writer win approach. I've cleaned it up.